README

```diff
@@ -50,20 +50,11 @@ murmurd [-supw <password>] [-ini <inifile>] [-fg] [v]
 Running Murmur on Mac OS X
 ==========================
 
-On Mac OS X, Murmur is available inside your Mumble application bundle. If you
-copied the Mumble program into your Applications directory, you should be able
-to run it by executing:
+Murmur is now distributed seperately from the Mumble client on Mac OS X.
```

Comment:
The doc should not refer to how it was previously. So, instead of saying "Murmur is now", you should simply state that Murmur is available from the Mumble web site, as "Static OS X Server".